src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -704,6 +698,7 @@ private void TrackNullableStateForAssignment(BoundExpression value, TypeSymbolWi
                     {
                         if (valueSlot > 0)
                         {
+                            bool isByRefTarget = IsByRefTarget(targetSlot);
```

Comment:
>bool isByRefTarget = IsByRefTarget(targetSlot); [](start = 28, length = 47)

It feels like we decided that ByRef-ness doesn't matter. Why do we keep it here?